packages/react-scripts/scripts/build.js

```diff
@@ -92,7 +92,11 @@ checkBrowsers(paths.appPath, isInteractive)
         );
         console.log(
           'To ignore, add ' +
-            chalk.cyan('// eslint-disable-next-line') +
+            chalk.cyan(
+              paths.useTSLint
```

Comment:
This message should mention both. For eslint use that and for tslint use that. Again, since there will be mixed projects 100%.

Reply:
I suggest we drop TSLint usage for now, so please revert this change.